### ISSUE TITLE
The Process of Training

### ISSUE THREAD

**Issue by YOKE** [doiken23/DeepLab_pytorch#2]
Thanks for your work! I'd like to train my datasets. Could your share the training scripts to make me know more about the details of the network. Thank you very much!

**Comment by doiken23** [comment]
Thanks BEYBB7!

I'll prepare the training script. So, please tell me your data source(jpg or png image, tiff image or numpy array)

**Comment by YOKE** [comment]
I‘d like to implement the network on LIP dataset, of which the image is jpg format and the label is png format. Thank you!

**Comment by doiken23** [comment]
I prepared dataset for **.png** in [this repository](https://github.com/doiken23/pytorch_toolbox).
